docs/source/usage.rst

```diff
@@ -3,18 +3,34 @@
 
 Running *MRIQC*
 ***************
-.. tip::
-     Try MRIQC online on `OpenNeuro <https://www.openneuro.org/>`_ - without
-     installation!
-
-MRIQC is a `BIDS-App <http://bids-apps.neuroimaging.io/>`_ [BIDSApps]_,
+*MRIQC* is a `BIDS-App <http://bids-apps.neuroimaging.io/>`_ [BIDSApps]_,
 and therefore it inherently understands the :abbr:`BIDS (brain
-imaging data structure)` standard [BIDS]_ and follows the
-BIDS-Apps standard command line interface::
+imaging data structure)` standard [BIDS]_.
+Before moving forward, please make sure to have read and understood
+*NiPreps*'s
+`introductory documentation <https://www.nipreps.org/apps/framework/>`__).
+
+Containerized execution with *Docker* and *Singularity*
```

Comment:
```suggestion
Containerized execution with *Docker* and *Singularity*/*Apptainer*
```

Reply:
I suggest referring to singularity/apptainer with this awkward slash for now. My understanding is that we will eventually call this only "apptainer", but right now things are in a bit of a transition state.